src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -16,6 +16,8 @@ namespace System.Diagnostics
 {
     public partial class Process : IDisposable
     {
+        private const string s_kfmclient = "kfmclient";
```

Comment:
I added this as const string because I was using the same string in two different place. 

strings "xdg-open" and "gnome-open" were not used in more than one place, so I didnt bother adding const string for them.

Reply:
that's fine. as a matter of practice, we don't have a rule. The C# compiler should intern strings (de dupe) so it isn't more efficient. It's more a matter of being possibly easier to maintain.

Reply:
I probably woudln't bother here.